src/components/TileHeader/TileHeader.tsx

```diff
@@ -19,8 +19,8 @@ export interface Props {
 export const TileHeader = ({ children, className, ...other }: Props) => {
   const componentClassName = clsx(styles['tile__header'], className, {});
   return (
-    <div className={componentClassName} {...other}>
+    <header className={componentClassName} {...other}>
```

Comment:
Not technically within the scope of this ticket, but as I was auditing our components I noticed that this should be using `<header>` instead of `<div>`.